.changeset/few-melons-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/cms-base": patch
+---
+
+Readme files for corresponding CMS elements.
```

Comment:
that's not a user-relevant change in the package (nor any package API change) - let's not add this changeset to avoid confusion